### PR TITLE
feat: upgrade gradle groovy to kotlin dsl

### LINKED ISSUE #2061: Feature: upgrade gradle grovvy to kotlin dsl

### PR DESCRIPTION
**Pull request by Aditya-gupta99**
Fixes #2061 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Aditya-gupta99** [comment]
@therajanmaurya Can you please review this PR.

File: mifosng-android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
-<!--    package="com.mifos.mifosxdroid">-->
+    <!--    package="com.mifos.mifosxdroid">-->
```
Comment (therajanmaurya) [inline mifosng-android/src/main/AndroidManifest.xml:4]: remove this we don't need this

File: mifosng-android/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -26,18 +26,16 @@
         <meta-data
             android:name="com.google.android.gms.version"
             android:value="@integer/google_play_services_version" />
-<!--        <meta-data-->
-<!--            android:name="io.fabric.ApiKey"-->
-<!--            android:value="facae47316396ae7954149cc408e25b7a37ab7ea" />-->
```
Comment (therajanmaurya) [inline mifosng-android/src/main/AndroidManifest.xml]: Open an issue to implement the firebase inplace of fabric, upgrade fabric to firebase

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by Aditya-gupta99** [comment]
@therajanmaurya I have done the required changes

**Review by therajanmaurya — APPROVED** [review]

**Pull request merged**